docs/docs/how-tos/nebari-local.md

```diff
@@ -173,18 +187,9 @@ certificate:
 Note the above snippet can be automatically provisioned in your configuration if you provided the `--ssl-cert-email` flag when you ran `nebari init`.
 
 :::note
-Let's Encrypt heavily rate limits their production endpoint and provisioning https certificates can often fail due to this limit.
+Let's Encrypt heavily rate limits their production endpoint.  In order to avoid throttling, Nebari's traefik deployments will store retrieved certificates for the duration of their validity in a mounted PVC at a default location `/mnt/acme-certificates/acme.json`.  In order to refresh the certificate before it is invalidated, you will need to delete the `acme.json` file then restart the Traefik deployment by deleting the existing pod and letting a new one spin up.  This may be necessary if you change the domain name of your Nebari deployment.
```

Comment:
I would break this, with only the new `In order to refresh...`part inside the notes, the rest could be added above it. I would also add a link if possible to what is the cert.json form the traefik+letsencrypt docs -- I think this would be enough https://doc.traefik.io/traefik/https/acme/#storage